src/providers/ldap/ldap_child.c

```diff
@@ -297,22 +300,24 @@ static krb5_error_code ldap_child_get_tgt_sync(TALLOC_CTX *memctx,
 
     if (!realm_str) {
         krberr = krb5_get_default_realm(context, &default_realm);
-        if (krberr) {
-            DEBUG(SSSDBG_OP_FAILURE, "Failed to get default realm name: %s\n",
-                      sss_krb5_get_error_message(context, krberr));
+        if (krberr != EOK) {
```

Comment:
Hi,

thank you for adding back the messages. I now only have a minor nitpick. I prefer to use `EOK` only for our own functions where we know they have `return EOK;`. For library calls like e.g. here `krb5_get_default_realm()` I'd prefer with the documented return values, so `0` in this case. I'm sure there are place where we do not handle this consistently but since I noticed it here, I wanted to add this comment.

Reply:
Good catch. I will fix this.

Reply:
Fixed.